SU-HSDO/suhumsci
# SHS-5779: Implementation: Testimonial Update

# READY FOR REVIEW

## Summary
New design for Testimonial

## Need Review By (Date)
09/24

## Urgency
medium

## Steps to Test
1. Go to this page `/components/testimonials` in Colorful and Traditional
3. Verify the `Testimonial` component looks according to the new [design]( https://www.figma.com/design/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?node-id=1900-2180&t=fXFnahJTU4zxg1a1-4) in both themes and all their color palettes
4. Please also test by adding the `hb-dark-pattern` class

FYI:
   - Links are different from what's in the design, they should use same color that they have been using in the entire site (check what's defined in `/components/text-area-typography`)
   - Be guided by what is in the design, not what is written.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208357272860401
